src/app/components/home/home.component.ts

```diff
@@ -47,7 +47,7 @@ export class HomeComponent implements OnInit, OnDestroy {
     this.swellService.getObject(name)
     .then( object => {
```

Comment:
swellService.getObject(objectId) requires a id as parameter, not a name. This code creates an object with id == name